.github/workflows/tox.yml

```diff
@@ -13,11 +13,11 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.6'
           - '3.7'
           - '3.8'
           - '3.9'
-    runs-on: ubuntu-20.04
+          - '3.10'
+    runs-on: ubuntu-latest
```

Comment:
I am still debating on being explicit and specify `ubuntu-22.04` to avoid breakages when GHA changes its latest flavor vs using `ubuntu-latest` to avoid updating N repos.

I can definitely see arguments on both sides. Either way, not a blocker for this PR and happy to go with the consensus. I wanted to bring it up in case we need to agree on a policy as we are periodically updating these workflows.